app/views/idv/session_errors/throttled.html.erb

```diff
@@ -1,8 +1,6 @@
 <%= render(
       'idv/shared/error',
-      heading: liveness_checking_enabled? ?
-        t('errors.doc_auth.throttled_heading_liveness') :
-        t('errors.doc_auth.throttled_heading'),
+      heading: t('errors.doc_auth.throttled_heading_liveness'),
```

Comment:
I think this got mixed up. The selfie string got stuck in here

Reply:
Yup. Put back.